experiments/migrations/0002_auto_20170904_1139.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-09-04 11:39
```

Comment:
Can we add a descriptive name for this migration?

Reply:
Sure, but I can't think of one. Any suggestions? I'm not sure why this migration hasn't been created in the original package. I haven't changes the models.

Reply:
can it be something like `0002_update_experiment_fields.py` ?

Reply:
👍